lib/clerk/rack_middleware_v2.rb

```diff
@@ -124,8 +124,12 @@ def call(env)
           return signed_out(env)  # malformed JSON authorization header
         end
 
-        token = verify_token(header_token)
-        return signed_in(env, token, header_token) if token
+        begin
+          token = verify_token(header_token)
+          return signed_in(env, token, header_token) if token
+        rescue JWT::ExpiredSignature, JWT::InvalidIatError
```

Comment:
(personal opinion) since `token` is used in `signed_in()` it's easier (for me) to read it like that instead of:
```ruby
begin
  token = verify_token(header_token)
rescue JWT::ExpiredSignature, JWT::InvalidIatError
  return interitial(...)
end

return signed_in(env, token, header_token) if token
```
In the above case we have to search 3 lines above for the definition of `token`. If they are defined as in my commit it's easier to read if you are context switching for a JS code base.
WDYT?

Reply:
Fine by me.